boards/shields/lcd_par_s035/Kconfig.defconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
is it needed to add
```c
if SHIELD_LCD_PAR_S035
```
?

Reply:
I do think this is needed- it follows the pattern of other shields: https://github.com/zephyrproject-rtos/zephyr/blob/76559f27fd6e9219516c9ee7deebbdf5b3116105/boards/shields/adafruit_2_8_tft_touch_v2/Kconfig.defconfig#L4